packages/image_picker/image_picker/README.md

```diff
@@ -41,6 +41,9 @@ It is no longer required to add `android:requestLegacyExternalStorage="true"` as
 **Note:** Images and videos picked using the camera are saved to your application's local cache, and should therefore be expected to only be around temporarily.
 If you require your picked image to be stored permanently, it is your responsibility to move it to a more permanent location.
 
+
+**Note:** Launching the image picker from an `Activity` with `launchMode: singleInstance` will always return `RESULT_CANCELED`. In this launch mode, new activities are created in a separate [Task][2]. As activities cannot communicate between tasks, the image picker activity cannot send back its eventual result to the calling activity. To work around this problem, consider using `launchMode: singleTask` instead.
```

Comment:
nit: break this up into multiple lines, similar to how we've formatted other blocks of text in this README.

Reply:
How would you want me to break up the lines? The way it is currently done in the README does not strike me as being consistent.
Should I limit lines to the 80 character count? Or should a start a newline when it makes sense for the flow of the text?

Option 1:
```
**Note:** Launching the image picker from an `Activity` with
`launchMode: singleInstance` will always return `RESULT_CANCELED`. In this
launch mode, new activities are created in a separate [Task][2]. As activities
cannot communicate between tasks, the image picker activity cannot send back
its eventual result to the calling activity. To work around this problem,
consider using `launchMode: singleTask` instead.
```

Option 2:
```
**Note:** Launching the image picker from an `Activity` with `launchMode: singleInstance` will always return `RESULT_CANCELED`.
In this launch mode, new activities are created in a separate [Task][2]. As activities cannot communicate between tasks, the image picker activity cannot send back its eventual result to the calling activity.
To work around this problem, consider using `launchMode: singleTask` instead.
```

Combining options 1 and 2:
```
**Note:** Launching the image picker from an `Activity` with
`launchMode: singleInstance` will always return `RESULT_CANCELED`.
In this launch mode, new activities are created in a separate [Task][2].
As activities cannot communicate between tasks, the image picker activity
cannot send back its eventual result to the calling activity.
To work around this problem, consider using `launchMode: singleTask` instead.
```

Reply:
I'd go with option 3 here. 

If you feel like making the rest of the readme fit this rule as well, that would be appreciated! You don't have to though.